content/documentation/staging/tutorial/01-prerequisites/02-install-istio.adoc

```diff
@@ -0,0 +1,15 @@
+---
+title: Install Istio
+weight: 12
+---
+:icons: font
+:sectlinks:
+
+Once you have your Kubernetes cluster ready, follow the https://istio.io/latest/docs/setup/getting-started/[Istio Getting Started, window="_blank"] to install and setup a demo profile that will be used in this tutorial.
+
+icon:bullhorn[size=1x]{nbsp} Determining ingress IP and ports and creating DNS entries will be necessary in the following steps.
+
+icon:bullhorn[size=1x]{nbsp} This tutorial uses https://istio.io/latest/docs/setup/platform-setup/minikube/[Minikube tunnel, window="_blank"] feature for external Ingress IP.
```

Comment:
Right. Minikube's tunnel is used to setup an External IP to the Ingress route as part of the prerequisites for the cluster setup.
Otherwise you can link a DNS entry with a public IP.
As this is platform dependend and it may vary in other environments I didn't enter in too many details but +1 to add a note.